logger/logger_test.go

```diff
@@ -148,3 +152,25 @@ func TestInitLogging(t *testing.T) {
 	// cleanup log file after test
 	os.RemoveAll(logFile)
 }
+
+func TestInitJaeger(t *testing.T) {
+	tracer, closer := InitJaeger("CSI-Driver")
+	defer closer.Close()
+	opentracing.SetGlobalTracer(tracer)
+
+	http.HandleFunc("/ping", func(w http.ResponseWriter, r *http.Request) {
+		span := StartSpanFromRequest(tracer, r)
+		defer span.Finish()
+
+		outboundHostPort, _ := os.LookupEnv("OUTBOUND_HOST_PORT")
+		ctx := opentracing.ContextWithSpan(context.Background(), span)
+		response, err := Ping(ctx, outboundHostPort)
```

Comment:
Looks like you are invoking Ping here. If it is needed only for test, we can move the Ping method here.